client/signup/config/steps.js

```diff
@@ -143,4 +143,13 @@ module.exports = {
 		providesDependencies: [ 'siteId', 'siteSlug', 'domainItem', 'themeSlugWithRepo' ],
 		delayApiRequestUntilComplete: true
 	},
+
+	'user-social': {
+		stepName: 'user-social',
+		props: {
+			headerText: i18n.translate( "Let's get started" ),
+			subHeaderText: i18n.translate( "Welcome to the wonderful WordPress.com community." )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Welcome to the wonderful WordPress.com community' )` *ES Score: 21*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).